src/applications/hca/actions.js

```diff
@@ -6,17 +6,31 @@ import { HCA_ENROLLMENT_STATUSES } from './constants';
 // flip the `false` to `true` to fake the endpoint when testing locally
 const simulateServerLocally = environment.isLocalhost() && false;
 
+// action types related to calling /health_care_applications/enrollment_status
 export const FETCH_ENROLLMENT_STATUS_STARTED =
   'FETCH_ENROLLMENT_STATUS_STARTED';
 export const FETCH_ENROLLMENT_STATUS_SUCCEEDED =
   'FETCH_ENROLLMENT_STATUS_SUCCEEDED';
 export const FETCH_ENROLLMENT_STATUS_FAILED = 'FETCH_ENROLLMENT_STATUS_FAILED';
+
+// action types related to calling GET /notifications/dismissed_statuses
+export const FETCH_DISMISSED_HCA_NOTIFICATION_STARTED =
+  'FETCH_DISMISSED_HCA_NOTIFICATION_STARTED';
+export const FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED =
+  'FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED';
+export const FETCH_DISMISSED_HCA_NOTIFICATION_FAILED =
+  'FETCH_DISMISSED_HCA_NOTIFICATION_FAILED';
+
+export const SET_DISMISSED_HCA_NOTIFICATION_STARTED =
+  'SET_DISMISSED_HCA_NOTIFICATION_STARTED';
```

Comment:
What about `DISMISSED_HCA_NOTIFICATION` for this? While it is a server call, I think having `STARTED` in it implies that you should look for the rest of the actions. And `DISMISSED_HCA_NOTIFICATION` is a pretty close description of what the user is doing, it looks like.

Reply:
How about `SET_DISMISSED_HCA_NOTIFICATION`? without the trailing `STARTED`? I think leading with `SET` pairs a little better with the `FETCH_DISMISSED...` actions

Reply:
Generally, I think action names that are initiated by a user action should reflect what the user did. A user isn't trying to set something, they're trying to dismiss the notification.

In this case, I don't think `SET_` is terrible or anything, unlike when we use it for actions that set a single property in the Redux state. Just thought `DISMISS` was a better description.